common/src/main/java/com/microsoft/identity/common/internal/ui/embeddedwebview/EmbeddedWebViewAuthorizationStrategy.java

```diff
@@ -57,11 +61,13 @@
      * @param request authorization request
      * @return generic AuthorizationResult
      */
-    public GenericAuthorizationResult requestAuthorization(final GenericAuthorizationRequest request) {
+    public GenericAuthorizationResult requestAuthorization(@Nullable final GenericAuthorizationRequest request) {
+        Logger.verbose(TAG, "Perform the authorization request with embedded webView.");
         loadStartUrl();
-        // TODO : Add state parameter from the AuthorizationRequest to the Intent parameter like below
-        // intent.putExtra(MicrosoftAuthorizationResult.REQUEST_STATE_PARAMETER, request.getState());
-        throw new UnsupportedOperationException("Not implemented yet.");
+        // requestAuthorization could not return the authorization result
+        // The activity result is set in AuthenticationActivity.setResult()
+        // And AuthenticationActivity in ADAL/MSAL is not moved into common
+        return null;
```

Comment:
If this cannot be returned from here, I suggest setting `Void` as the return type in:
`... extends AuthorizationStrategy<Void, GenericAuthorizationResult>` although this also means that inheritors of this class must `@Override` this implementation if they want a return value....

Does this method need to be abstract?